src/meta/protos/proto/table.proto

```diff
@@ -57,6 +58,16 @@ message TableIdent {
   uint64 seq = 2;
 }
 
+//message TableStorage {
+//  oneof storage {
+//    S3StorageConfig s3 = 1;
+//    FsStorageConfig fs = 2;
+//    GcsStorageConfig gcs = 3;
+//  }
+//}
+
```

Comment:
You can just remove these lines unless you'd like to have a monument for the code you've written. 🤔